docs/reference/query-dsl/match-query.asciidoc

```diff
@@ -142,21 +142,6 @@ out of the analyzed text. For example:
 }
 --------------------------------------------------
 
-Since `match_phrase` is only a `type` of a `match` query, it can also be
-used in the following manner:
-
-[source,js]
---------------------------------------------------
-{
-    "match" : {
-        "message" : {
-            "query" : "this is a test",
-            "type" : "phrase"
-        }
-    }
-}
---------------------------------------------------
-
```

Comment:
I would definitely remove those complete examples. just wondering if it makes sense to just mention that we changed something, maybe (optional) also explaining why.